init.c

```diff
@@ -100,7 +103,17 @@ utility_hook(Node *parsetree,
 
 	PG_TRY();
 	{
-		standard_ProcessUtility(parsetree, queryString, context, params, dest, completionTag);
+		/* If there's another hook in the chain, pass on the action */
+		/* and hope/pray that it passes it along eventually to the */
```

Comment:
I was going to say that the implementation described here http://www.postgresql.org/docs/9.4/static/sql-createeventtrigger.html would be over-ridden, but I'd have to read deeper to find out for sure. The over-ride in this code block is just capturing all `CREATE EVENT TRIGGER` commands, but farther down the code is still populating the `pg_event_trigger` system table, so the default implementation might still be perfectly happy.